SharedProjects/BabylonExport.Entities/BabylonScene.cs

```diff
@@ -218,5 +218,17 @@ public void Prepare(bool generateDefaultLight = true, bool generateDefaultCamera
                 animationGroups = null;
             }
         }
+
+        internal bool PackIndexArrays()
```

Comment:
nit: Naming is inconsistent with the one in mesh. Maybe both should start with `Try` or maybe don't use this prefix because this code isn't actually following the `Try` prefix pattern?